index.bs

```diff
@@ -36,9 +36,8 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
             text: worker; url: worker
     urlPrefix: infrastructure.html
         type: dfn
+            text: structuredclone; url: structuredclone
```

Comment:
You should be able to not do this if you use `<a abstract-op>StructuredClone</a>(...)` instead of `[=StructuredClone=](...)`. This will also ensure it shows up in your indices with the correct casing.